src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1433,8 +1448,9 @@ void RadioSpinel<InterfaceType, ProcessContextType>::HandleTransmitDone(uint32_t
                                                                         const uint8_t *   aBuffer,
                                                                         uint16_t          aLength)
 {
-    otError         error  = OT_ERROR_NONE;
-    spinel_status_t status = SPINEL_STATUS_OK;
+    otError         error        = OT_ERROR_NONE;
+    spinel_status_t status       = SPINEL_STATUS_OK;
+    bool            framePending = false;
     spinel_ssize_t  unpacked;
 
     VerifyOrExit(aCommand == SPINEL_CMD_PROP_VALUE_IS && aKey == SPINEL_PROP_LAST_STATUS, error = OT_ERROR_FAILED);
```

Comment:
not relative to this PR, but the doxygen for `SPINEL_PROP_LAST_STATUS` should be updated? @bukepo 

https://github.com/openthread/openthread/blob/6de8d778b784d2ef6af409e6b85d511cbc8fc3b5/src/lib/spinel/spinel.h#L1149-L1163

Reply:
revise a bit - also relative to this PR. @brianljt

Reply:
Please check the discussion here. https://github.com/openthread/openthread/pull/4976#discussion_r433343963

The TxDone info is appended after `SPINEL_PROP_LAST_STATUS`